### PR TITLE
SOMMELIER-4 

### PR DESCRIPTION
**Pull request by Maximus135**
Pull after fixes.

### CODE REVIEW

File: App/components/SearchInput/input.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <Styled.StyledView editable={editable}>
+      <Search
+        width={theme.searchSizes[size]}
+        height={theme.searchSizes[size]}
+      />
+      <Styled.StyledInput
+        defaultValue={defaultValue}
+        placeholder={placeholder}
+        editable={editable}
+        size={size}
```
Comment (StanislavMyakishev) [inline App/components/SearchInput/input.tsx:45]: Missed that in the first review, but it seems that we can spread whole props obj and pass on it to the inner input

**Review by StanislavMyakishev — CHANGES_REQUESTED** [review]

File: App/components/SearchInput/input.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            <Close
+              width={theme.inputFontSizes[size]}
+              height={theme.inputFontSizes[size]}
+            />
+          </TouchableOpacity>
+        </Styled.ViewClose>
+      )}
+    </Styled.StyledView>
+  )
+}
```
Comment (Mixxx-a) [inline App/components/SearchInput/input.tsx:63]: add `export default SearchInput`

**Review by StanislavMyakishev — APPROVED** [review]

File: App/theme/styled.d.ts [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { boxSizes } from './boxSizes'
 import { colors } from './Colors'
 import { borders } from './borders'
 import { fontSizes } from './FontSizes'
+import { inputSizes, searchSizes, inputFontSizes } from './inputSearchSizes'
```
Comment (Mixxx-a) [inline App/theme/styled.d.ts]: Should './inputSearchSizes' be capitalized like ./'**I**nputSearchSizes' (change file name)?

File: App/components/SearchInput/input.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react'
+import {
+  NativeSyntheticEvent,
+  TextInputChangeEventData,
+  TouchableOpacity,
+} from 'react-native'
+import SvgUri from 'react-native-svg-uri'
```
Comment (Mixxx-a) [inline App/components/SearchInput/input.tsx:7]: don't use SvgUri, delete this import

File: App/components/SearchInput/input.tsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+type Props = {
+  defaultValue?: string
+  placeholder?: string
+  editable: boolean
+  size: keyof typeof theme.inputSizes
+  maxLength?: number
+  value?: string
+  onChange: (e: NativeSyntheticEvent<TextInputChangeEventData>) => void
+  clearButtonMode: boolean
```
Comment (Mixxx-a) [inline App/components/SearchInput/input.tsx]: Shouldn't all props be additional (with :?) (see trello description)?

**Review by Mixxx-a — CHANGES_REQUESTED** [review]

**Pull request closed without merging**